mongo/client_test.go

```diff
@@ -344,4 +344,49 @@ func TestClient(t *testing.T) {
 			})
 		}
 	})
+	t.Run("serverAPI version", func(t *testing.T) {
+		t.Run("success", func(t *testing.T) {
+			ServerAPIOptions := options.ServerAPI().SetServerAPIVersion(options.ServerAPIVersion1)
```

Comment:
nit: local variables shouldn't be capitalized. Change to a lowercase "s".

Reply:
Mondays 🤦 . Done for here and elsewhere.